src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2111,13 +2111,20 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         //      actually fail. We need a way to gracefully fallback.
         _renderer->TriggerFontChange(newDpi, _desiredFont, _actualFont);
 
+        const auto actualFontName = _actualFont.GetFaceName();
+        const auto desiredFontName = _desiredFont.GetFaceName();
+
+        auto caseInsensitiveEquality = [](wchar_t a, wchar_t b) { return ::towlower(a) == ::towlower(b); };

```

Comment:
Worried that this uses the user's locale for a not-locale-specific font name. Maybe we can use `CompareStringOrdinal` with `bIgnoreCase=TRUE` to get around that?